pkgs/desktops/gnome-3/core/gnome-shell/default.nix

```diff
@@ -13,16 +13,13 @@ let
 
 in stdenv.mkDerivation rec {
   name = "gnome-shell-${version}";
-  version = "3.28.3";
+  version = "3.30.0";
 
   src = fetchurl {
     url = "mirror://gnome/sources/gnome-shell/${gnome3.versionBranch version}/${name}.tar.xz";
-    sha256 = "0xm2a8inj2zkrpgkhy69rbqh44q62gpwm4javzbvvvgx0srza90w";
+    sha256 = "0fgjy5z3cmnsh62rjgy24ppws1g39n9kis7x9s28acfdb1r2j7k2";
   };
 
-  # Needed to find /etc/NetworkManager/VPN
-  mesonFlags = [ "--sysconfdir=/etc" ];
```

Comment:
I wonder why was this removed?

Reply:
The build tried to install autostart files there. The networkmanager code is gone from upstream.

Reply:
Left something in the commit message, but should've marked it as squash. The relevant bit I guess:
> The flag should no longer be needed for networkmanager detection as the code
from 3.22 is no longer there (js/ui/components/networkAgent.js:768 @3.22.0).